src/Components/Tile.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Tile = ({title, children,image,description}) =>{
```

Comment:
Je code reageert hier niet op de aan/af-wezigheid van een image. Je kunt zoals je bij mijn PR suggereerde inderdaad een image toevoegen wanneer er een aanwezig is met `{image && <img src={image} alt={description}}`.